docs/about/index.md

```diff
@@ -0,0 +1,86 @@
+
+
+```csharp
+using System;
+
+class Program
+    {
+        static void Main(string[] args)
+        {
+        	Console.WriteLine("Hello World");
+        }
+    }
+```
+
+Try .NET
+--------
+
+Try out .NET Core. It is the quickest and easiest way to try out .NET, on Windows, macOS or Linux. You can be running .NET Core on your machine in 5 minutes. Try it now or watch it below. TODO: Add link here.
+
+TODO: Animation or video of those 5 mins. 
+
+Use .NET for Any App
+--------------------
+
+You can build any app with .NET. There are many highly capable and popular .NET solutions, app types and libraries that you can use, provided by various companies and individuals.
+
+* Web
+* Mobile
+* Cloud
+* Desktop
+* Games
+* IoT
+* Open Source
+
+TODO: Add detail pages for all of these and then link to those.
+
+Learning about .NET
+-------------------
+
+You can learn about .NET, the best ways to use it and what kind of apps you can build with it. 
+
+.NET
+----
+
+.NET is a general purpose development platform. It is used for many kinds of apps and workloads by using the many [.NET Products](products.md) that are available. It has several key features that make it easy and efficient to build great applications, including automatic memory management and modern programming languages. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
+
+- Learn about [.NET](concepts)
+- Learn about C#, F# and Visual Basic (each language needs a link)
+- Create portable libraries (link here)
+- Browse the [.NET API Library](../../api)
+- See great .NET apps and games (link here)
+- Participate in the .NET community (link here)
+
+.NET Core
+---------
+
+.NET Core is used to build device, web, cloud and embedded/IoT apps. It is [open source](https://github.com/dotnet/core) and cross-platform, supporting Windows, macOS and Linux. [ASP.NET Core](http://docs.asp.net/) is the most popular workload for .NET Core. You can use it to build web apps and services, for on-premises and cloud deployment. You can also use .NET Core to build tools, utilities and cloud worker apps.
+
+- Learn about [.NET Core](core-concepts)
+- Learn about [ASP.NET Core](http://docs.asp.net/)
+- Get started building [.NET Core apps](http://dot.net/core)
+
+.NET Framework
+--------------
+
+The .NET Framework is used to build apps for Windows and Windows Server. You can use it to build rich user interfaces with Windows Presentation Framework (WPF) and Windows Forms. You quickly build server apps with ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF). It is integrated into Visual Studio, which provides many rich designer experiences for building both client and server apps. It is the best choice for writing apps for Windows.
+
+- Learn about the [.NET Framework](https://msdn.microsoft.com/library/w0x726c2.aspx)
+- Get started building [Windows apps](https://dot.net)
+
+Xamarin
```

Comment:
OK, this may be changing, but...

Should this be "Xamarin Platform" (which was the product name), and not "Xamarin", which was the company name?